WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -20,6 +20,50 @@ public class ReaderScreen: ScreenObject {
         )
     }
 
+    public func openTheLastPostInApp() {
+        getTheLastPost().tap()
+    }
+
+    public func openTheLastPostInSafari() {
+        getTheLastPost().buttons["More"].tap()
+        app.buttons["Visit"].tap()
+    }
+
+    public func getTheLastPost() -> XCUIElement {
+        let postPosition = app.cells.count - 1
+        let post = app.cells.element(boundBy: postPosition)
+
+        scrollDownUntilElementIsHittable(element: post)
+        return post
+    }
+
+    private func scrollDownUntilElementIsHittable(element: XCUIElement) {
+        var loopCount = 0
+        while !element.waitForIsHittable(timeout: 3) && loopCount < 10 {
+            loopCount += 1
+            app.swipeUp(velocity: .fast)
+            print(loopCount)
+        }
+    }
+
+    public func postContentEquals(expected: String) -> Bool {
```

Comment:
Another naming nitpick. Following the ["Omit needless words" guideline](https://www.swift.org/documentation/api-design-guidelines/#naming)

```suggestion
    public func postContentEquals(_ content: String) -> Bool {
```

or if you prefer:

```suggestion
    public func postContentEquals(_ expected: String) -> Bool {
```

<img width="779" alt="Screen Shot 2021-11-24 at 4 02 06 pm" src="https://user-images.githubusercontent.com/1218433/143177918-5c06c9e7-6fec-4498-ac33-f16df268a2fd.png">

Reply:
Yeah, way better!